dm/dm/ctl/master/binlog.go

```diff
@@ -90,11 +125,23 @@ func newBinlogRevertCmd() *cobra.Command {
 // FIXME: implement this later.
```

Comment:
suddenly find that we can remove this comment now

Reply:
ok